tools/loader/ImageClassifier.cpp

```diff
@@ -109,9 +123,11 @@ llvm::cl::opt<std::string> modelInputName(
     llvm::cl::cat(imageLoaderCat));
 } // namespace
 
-/// Loads and normalizes all PNGs into a tensor in the NCHW 3x224x224 format.
+/// Loads and normalizes all PNGs into a tensor in the NHWC format with the
+/// requested channel ordering.
 void loadImagesAndPreprocess(const llvm::cl::list<std::string> &filenames,
-                             Tensor *result, ImageNormalizationMode normMode) {
+                             Tensor *result, ImageNormalizationMode normMode,
+                             ImageFormat format) {
```

Comment:
nit: `imageMode` and `imageFormat` are global for this file, so there's no need to pass them to this function.

Reply:
I think it's not necessary a good practice to use global variables extensively. My preference would be to limit the usage of global variables to the top level (in `main`) and pass them as function parameter. It is consistent with what was already already done.